detect/buildpacks.go

```diff
@@ -116,13 +117,19 @@ var _ = DetectDescribe("Buildpacks", func() {
 	})
 
 	Describe("dotnet-core", func() {
-		It("makes the app reachable via its bound route", func() {
-			Expect(cf.Cf("push", appName, "-m", DEFAULT_MEMORY_LIMIT, "-p", assets.NewAssets().DotnetCore).Wait(Config.DetectTimeoutDuration())).To(Exit(0))
-
-			Eventually(func() string {
-				return helpers.CurlAppRoot(Config, appName)
-			}).Should(ContainSubstring("Hello from dotnet-core"))
-		})
+		for _, stack := range Config.GetStacks() {
+			It(fmt.Sprintf("[%s] makes the app reachable via its bound route", stack), func() {
```

Comment:
`stack` needs to be assigned to a local copy inside of the for loop or else "the It closure will capture the mutating loop variable and all the specs will run against the last element in the loop." (re: [Dynamically Generating Specs](https://onsi.github.io/ginkgo/#dynamically-generating-specs))